core/commands/ls.go

```diff
@@ -47,7 +47,7 @@ const (
 
 var LsCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "List directory contents for Unix filesystem objects.",
+		Tagline: "List directory contents for Unix filesystem objects. (Only IPFS paths)",
```

Comment:
I think this is more likely to confuse than it is to help. Seeing this, my immediate reaction would be "Only IPFS paths? What other paths could this list?"